workflows/argo/chgres-cube.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: chgres-cube
+spec:
+  arguments:
+    parameters:
+    - name: restarts
+    - name: date
+    - name: source_resolution
+      value: C48
+    - name: target_resolution
+      value: C384
+    - name: tracers
+      value: '"sphum","liq_wat","o3mr","ice_wat","rainwat","snowwat","graupel","sgs_tke"'
+    - name: vcoord_file
+    - name: reference_data
+      value: gs://vcm-ml-raw-flexible-retention/2023-02-24-chgres-cube-forcing-data/2023-02-24-chgres-cube-forcing-data
+    - name: destination_root
+    - name: mpi_tasks
+      value: "6"
+    - name: memory
+      value: "25Gi"
```

Comment:
It can be relevant if we ever target grids of finer resolution than C384 (I have at least done that on Gaea before), but I don't envision us doing that anytime soon (I would need to upload additional reference data).  Depending on the resolution it could also require changing to a different kind of cluster node, which this doesn't handle currently.  I can remove it for now.